Documentation/network/servicemesh/mutual-authentication/mutual-authentication.rst

```diff
@@ -0,0 +1,107 @@
+
+    Cilium's encryption features,  :ref:`encryption_wg` and :ref:`encryption_ipsec`, can be enabled 
+    to automatically create and maintain encrypted connections between Pods.
+
+To address the challenge of identity verification in dynamic and heterogeneous environments, mutual authentication requires a framework secure identity verification for distributed systems.
+
+.. Note::
+
+    To learn more about the the Mutual Authentication architecture for the Cilium Service Mesh, read the `CFP <https://github.com/cilium/design-cfps/blob/main/cilium/CFP-22215-mutual-auth-for-service-mesh.md>`_.
+
+Identity Management
+###################
+
+In Cilium's current mutual authentication support, identity management is provided through the use of SPIFFE (Secure Production Identity Framework for Everyone).
+
+SPIFFE benefits
+---------------
+Here are some of the benefits provided by `SPIFFE <https://spiffe.io/>`_ :
+
+- Trustworthy identity issuance: SPIFFE provides a standardized mechanism for issuing and managing identities. 
+  It ensures that each service in a distributed system receives a unique and verifiable identity, even in dynamic environments where services may scale up or down frequently.
+- Identity attestation: SPIFFE allows services to prove their identities through attestation. 
+  It ensures that services can demonstrate their authenticity and integrity by providing verifiable evidence about their identity, like digital signatures or cryptographic proofs.
+- Dynamic and scalable environments: SPIFFE addresses the challenges of identity management in dynamic environments. 
+  It supports automatic identity issuance, rotation, and revocation, which are critical in cloud-native architectures where services may be constantly deployed, updated, or retired.
+
+Cilium and SPIFFE
+-----------------
+
+SPIFFE provides an API model that allows workloads to request an identity from a central server. In our case, a workload means the same thing that a Cilium Security Identity does - a set of pods described by a label set. 
+A SPIFFE identity is a subclass of URI, and looks something like this: ``spiffe://trust.domain/path/with/encoded/info``.
+
+There are two main parts of a SPIFFE setup:
+
+- A central SPIRE server, which forms the root of trust for the trust domain.
+- A per-node SPIRE agent, which first gets its own identity from the SPIRE server, then validates the identity requests of workloads running on its node.
+
+When a workload wants to get its identity, usually at startup, it connects to the local SPIRE agent using the SPIFFE workload API, and describes itself to the agent.
+
+The SPIRE agent then checks that the workload is really who it says it is, and then connects to the SPIRE server and attests that the workload is requesting an identity, and that the request is valid. 
+
+The SPIRE agent checks a number of things about the workload, that the pod is actually running on the node it’s coming from, that the labels match, and so on. 
```

Comment:
Curly apostrophe again (`it’s`), please do a search on your content (the search function in my browser doesn't make the distinction between `’` and `'` so it's hard to check on GitHub).